lms/templates/courseware/progress.html

```diff
@@ -80,6 +80,13 @@ <h2 class="title">${_("We're working on it...")}</h2>
                     <p class="copy">${_("We're creating your certificate. You can keep working in your courses and a link to it will appear here and on your Dashboard when it is ready.")}</p>
                   </div>
                   <div class="msg-actions"></div>
+                  %elif not is_id_verified or is_unverified:
+                   <div class="msg-content">
+                    ## Translators: This message appears to users when the users have not completed identity verification.
+                    <h2 class="title">${_("Certificate unavailable")}</h2>
+                    <p class="copy">${_("Your certificate is not issued because you do not have a current verified identity with edX. ")} <a href="${reverse('verify_student_reverify')}"> ${_("Verify your identity now.")}</a></p>
```

Comment:
How about:

You have not received a certificate because you do not have a current `platform` verified identity. Verify your identity now.

I hesitate to remove "with edX" because "you do not have a current verified identity" is a bit broad. The learner has a verified identity, just not one on edX. :-)